src/applications/disability-benefits/all-claims/content/traumaticEventDetails.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+export const eventDetailsPageTitle = 'Details about your traumatic event';
+export const eventDetailsPrompt =
+  'Briefly describe what happened during your traumatic event in the military.';
+export const eventDetailsHint =
+  'We don’t need to know the names of people that were involved in the traumatic events. If you do share any names, we won’t contact an of those individuals.';
```

Comment:
> we won’t contact **an** of those individuals.

typo? "an" ➡ "any"

Reply:
Appreciate you catching that. An unfortunate byproduct of working out of Figma without access to Dev Mode. I'll resolve in a subsequent PR.